class_person.rb

```diff
@@ -21,6 +22,10 @@ def correct_name
     name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
+
```

Comment:
- [ ] Great job working so far for adding the rental but I think it would be great if you add `book, date` as parameters in add rental method here, please don't forget to pass the person itself when creating the new rental achieving the 'has-many' association` between the book and the person so kindly pass them then you can create the new rental 💯✅⭐🙏🏻

you can do something like this 


```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```